### PR TITLE
[CDF-11405] Only treat 200/201/202 as successfull response

### PR DESCRIPTION
**Pull request by erlendvollset**

### CODE REVIEW

**Review by sanderland — APPROVED** [review]

**Comment by sanderland** [comment]
the api returns 202?
 exceptions = [CogniteAPIError(b'{\n  \n}', 202, '76c33b95-590d-94bb-a97e-a01dd98cd91b')]

File: cognite/client/_api_client.py [hunk 1]
```diff
@@ -707,8 +707,8 @@ def _is_single_identifier(ids, external_ids):
         return single_id or single_external_id
 
     @staticmethod
-    def _status_is_valid(status_code: int):
-        return status_code < 400
+    def _status_ok(status_code: int):
+        return status_code in {200, 201}
```
Comment (sanderland) [inline cognite/client/_api_client.py:711]:
```suggestion
         return status_code in {200, 201, 202}
```

**Review by sanderland — COMMENTED** [review]
202 is documented

**Comment by erlendvollset** [comment]
@sanderland thanks. updated now 👍

**Review by sanderland — APPROVED** [review]

**Pull request merged**